### PR TITLE
Fixed deletion change

### PR DESCRIPTION
**Pull request by NejcZdovc**
PR based on #175.

Currently blocked by #176, because when you delete key action positioning is messed up. When #176 is done, this PR should work correctly.

### CODE REVIEW

**Comment by fjozsef** [comment]
@NejcZdovc Do you want me to squash the commits or add them both?

File: src/store/reducers/keymap.ts [hunk 1]
```diff
@@ -90,16 +90,24 @@ export default function (state = initialState, action: Action): KeymapState {
...
+            });
 
             // If deleted one is default set default keymap to the first on the list of keymaps
             if (isDefault && filtered.length > 0) {
                 filtered[0].isDefault = true;
             }
 
+            // Check for the deleted keymap in other keymaps
+            newState = filtered.map((keymap: Keymap) => {
+                changedKeymap =  new Keymap();
```
Comment (fjozsef) [inline src/store/reducers/keymap.ts]: Remove extra space before new

Comment (NejcZdovc) [inline src/store/reducers/keymap.ts]: fixed

**Comment by NejcZdovc** [comment]
Commits are squashed, so you can merge it

**Pull request merged**